android/app/build.gradle

```diff
@@ -137,10 +137,12 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-vector-icons')
```

Comment:
@andangrd could you please revert this file? These changes are unnecessary. It's already there, using `implementation` instead of `compile`.

```java
    implementation project(':appcenter')	
    implementation project(':appcenter-analytics')	
    implementation project(':appcenter-crashes')	
    implementation project(':react-native-vector-icons')
```